examples/protected_base/construct.yaml

```diff
@@ -12,13 +12,18 @@ specs:
   - python
   - conda
 
+freeze_base:
+  conda: {}
+
 extra_envs:
-  default:
+  env1:
     specs:
       - python
       - pip
       - conda
+    freeze_env:
+      conda:
+        "message": "This environment is frozen."
 
 extra_files:
```

Comment:
Currently, your `construct.yaml` file is the one that's broken. It would be much nicer if you committed the one that's working to the repository and then add this later for the negative test case.